cowprotocol/cowswap
# Added page titles for GA tracking

# Summary

This PR adds page titles which will also be available in the GA and is important for better tracking
The title pattern is `{Current page title} - {APP_TITLE}` where `APP_TITLE` is `CoW Swap | The smartest way to trade cryptocurrencies` as before

![Screenshot from 2022-09-23 15-28-22](https://user-images.githubusercontent.com/34926005/191970974-6026a911-8d6a-41cb-ae5f-d9ec7b689c5d.png)

# To test
- make sure you have the GA extension on
- open dev tools -> Console
- filter by `page view`
- with each page change make sure you see the right parameter passed as `dt` in `Shared parameters`

![Screenshot from 2022-09-23 17-46-06](https://user-images.githubusercontent.com/34926005/192001195-6d4ac757-d084-4731-8a79-40798f34f3cf.png)
